static/src/js/pages/ManageCollectionAssociation/ManageCollectionAssociation.jsx

```diff
@@ -0,0 +1,448 @@
+import { useLazyQuery, useMutation } from '@apollo/client'
+import React, {
+  useCallback,
+  useEffect,
+  useState
+} from 'react'
+import { useNavigate, useParams } from 'react-router'
+import { useSearchParams } from 'react-router-dom'
+import pluralize from 'pluralize'
+import { Alert } from 'react-bootstrap'
+import conceptTypeQueries from '../../constants/conceptTypeQueries'
+import getConceptTypeByConceptId from '../../utils/getConceptTypeByConcept'
+import errorLogger from '../../utils/errorLogger'
+import parseError from '../../utils/parseError'
+import Page from '../../components/Page/Page'
+import ErrorBanner from '../../components/ErrorBanner/ErrorBanner'
+import LoadingBanner from '../../components/LoadingBanner/LoadingBanner'
+import toLowerKebabCase from '../../utils/toLowerKebabCase'
+import Button from '../../components/Button/Button'
+import Table from '../../components/Table/Table'
+import conceptTypes from '../../constants/conceptTypes'
+import EllipsisLink from '../../components/EllipsisLink/EllipsisLink'
+import EllipsisText from '../../components/EllipsisText/EllipsisText'
+import { DELETE_ASSOCIATION } from '../../operations/mutations/deleteAssociation'
+import CustomModal from '../../components/CustomModal/CustomModal'
+import useAccessibleEvent from '../../hooks/useAccessibleEvent'
+import useNotificationsContext from '../../hooks/useNotificationsContext'
+
+/**
+ * Renders a ManageCollectionAssociation component
+ *
+ * @component
+ * @example <caption>Render a ManageCollectionAssociation</caption>
+ * return (
+ *   <ManageCollectionAssociation />
+ * )
+ */
+const ManageCollectionAssociation = () => {
+  const { conceptId } = useParams()
+
+  const navigate = useNavigate()
+  const { addNotification } = useNotificationsContext()
+
+  const [fetchedDraft, setFetchedDraft] = useState()
+  const [loading, setLoading] = useState(false)
+  const [tableLoading, setTableLoading] = useState()
+  const [error, setError] = useState()
+  const [searchParams, setSearchParams] = useSearchParams()
+  const [collectionConceptIds, setCollectionConceptIds] = useState([])
+  const [showDeleteModal, setShowDeleteModal] = useState(false)
+
+  const limit = 20
```

Comment:
If you look at searchpage.jsx, Trevor's solution for this was to destructure limit in the props and then make a default prop type of 25.

Reply:
Two different use cases here. For searchPage.jsx if we want to override that limit we can pass it into the component. Since this is the only place where this manageColletcionAssociation is going to be used, I do not think moving the limit to props makes sense.